Chap_API_Struct.tex

```diff
@@ -3699,12 +3699,12 @@ \subsection{Request-related attributes}
 
 %
 \declareAttribute{PMIX_JOB_TERM_STATUS}{"pmix.job.term.status"}{pmix_status_t}{
-Status to be returned upon job termination.
+Status returned by job upon its termination.
 }
 
```

Comment:
Do we document which API's might return PMIX_JOB_TERM_STATUS or PMIX_PROC_STATE_STATUS?   The descriptions are much improved, but in the version of the document I am looking at, I don't see when I would look for these?

Reply:
It's because they are returned in an event notification when the job or proc terminates, not from an API. I've been scratching my head a bit over this since you opened the cited issue as it isn't clear to me how to handle event-based attributes and values. They aren't really associated with any API. I was going to describe some of them in the tool chapter, but they really aren't tool-specific either - e.g., the ULFM folks use notifications of proc termination.

Perhaps the best approach is to expand the event notification chapter and cover them there? We could explain what attributes/values we expect the host to provide (e.g., job and proc termination events), how one requests that service, etc.

Would that make sense?

Reply:
I described how these values are communicated in the attribute definition itself - see what you think.